web/tasks/DART.md

```diff
@@ -1,7 +1,8 @@
+
 ---
-title: 'DART'
-type: Structure-to-Text
-motivation: TODO.
+title: DART
+type: Table-to-Text
```

Comment:
We decided to group ToTTo and DART under Structure to Text to avoid confusion between the two. Would you be able to change it back?